qai_plugin.py

```diff
@@ -153,6 +153,13 @@ def on_privmsg(self, *args, **kwargs):
 
         if sender.startswith("NickServ!"):
             self.__handleNickservMessage(msg)
+            return
+        if not msg.startswith ("!"):
+            if not sender in self.timers:
+                self.timers[sender] = 16
+            if time.time() - self.timers[sender] > 15:
+                self.bot.privmsg(kwargs['mask'].nick, "This is an official bot run by Forged Alliance Forever. Replies to this bot are not monitored. If you need to talk to a moderator, find someone with their name in white at the top of the player list. If there are none online at this time then ask someone in chat to help you.")
```

Comment:
"find someone with their name in white at the top of the player list"
This makes assumptions about the user interface, but it may change, the user may have a different style installed, or he may use a different client

Reply:
QAI already uses this construct in the spam_protect though? @ https://github.com/FAForever/QAI/blob/master/qai_plugin.py#L517